### PR TITLE
SUBWORKFLOW: Include call_structural_variants

### PR DESCRIPTION
**Pull request by projectoriented**
## PR checklist
This PR sets the stage for incoming SV workflows with sv caller manta.
- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker -stub`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/modules.config [hunk 1]
```diff
@@ -126,8 +126,8 @@ process {
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
         ]
     }
-    // CALL_SV_MANTA
-    withName: MANTA {
+    // CALL_STRUCTURAL_VARIANTS
+    withName: ".*CALL_STRUCTURAL_VARIANTS:CALL_SV_MANTA:MANTA" {
```
Comment (ramprasadn) [inline conf/modules.config:130]: I am not very familiar with this syntax, but does including the subworkflow name make it more specific to the process?

**Review by ramprasadn — APPROVED** [review]
LGTM 👍🏻

File: conf/modules.config (hunk 1 above)
Comment (projectoriented) [reply conf/modules.config:130]: Yup! It's helpful to use this syntax for the selector when you want multiple processes within the same subworkflow. For this one specifically, I thought it's better to be explicit than implicit 🤷 . We can change laterz

Comment (jemten) [reply conf/modules.config:130]: Cool. What does the .* do?

**Review by jemten — APPROVED** [review]
Very nice!

File: conf/modules.config (hunk 1 above)
Comment (projectoriented) [reply conf/modules.config:130]: @jemten The `.*` globs the preceding stuff so this:
`NFCORE_RAREDISEASE:RAREDISEASE:CALL_STRUCTURAL_VARIANTS:CALL_SV_MANTA:MANTA`

Comment (jemten) [reply conf/modules.config:130]: Ah!

Comment (ramprasadn) [reply conf/modules.config:130]: Sure! Makes sense 👍🏻

**Pull request merged**